provisioner/salt-masterless/provisioner.go

```diff
@@ -11,8 +11,10 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"regexp"
 	"strings"
 
+	"github.com/hashicorp/go-getter"
```

Comment:
```suggestion
	"github.com/hashicorp/go-getter/v2"
```

Reply:
Good point! I've updated the implementation to use go-getter/v2, did the go mod/vendor dance.

Reply:
Nice!! Thanks!